modules/migration-isolating-dns-domain-of-target-cluster-from-clients.adoc

```diff
@@ -1,26 +1,27 @@
 // Module included in the following assemblies:
 //
 // * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-mtc.adoc
 
 [id="isolating-dns-domain-of-target-cluster-from-clients_{context}"]
 == Isolating the DNS domain of the target cluster from the clients
 
 You can allow the clients' requests sent to the DNS domain of the source cluster to reach the DNS domain of the target cluster without exposing the target cluster to the clients.
 
 .Procedure
-. Place an exterior network component, such as an application load balancer or a reverse proxy, between the clients and the OpenShift Container Platform 4 cluster.
+. Place an exterior network component, such as an application load balancer or a reverse proxy, between the clients and the {product-title} {product-version} cluster.
```

Comment:
You refer to it as a target cluster in the introduction. Use "target cluster" here for consistency.